tests/foreman/virtwho/utils.py

```diff
@@ -142,6 +145,59 @@ def get_virtwho_status():
         return 'undefined'
 
 
+def get_configure_id(name):
+    """Return the configure id by hammer.
+    :param str name: the configure name you have created.
+    :raises: VirtWhoError: If failed to get the configure info by hammer.
+    """
+    config = VirtWhoConfig.info({'name': name})
+    if 'id' in config['general-information']:
+        return config['general-information']['id']
+    else:
+        raise VirtWhoError("No configure id found for {}".format(name))
+
+
+def get_configure_command(config_id, org=1):
+    """Return the deploy command line based on configure id.
+    :param str config_id: the unique id of the configure file you have created.
+    :param int org: the satellite organization id.
+    """
+    return (
+        "hammer virt-who-config deploy --id {} --organization-id {}"
+        .format(config_id, org)
+    )
+
+
+def get_configure_file(config_id):
+    """Return the configure file full name in /etc/virt-who.d
+    :param str config_id: the unique id of the configure file you have created.
+    """
+    return (
+        "/etc/virt-who.d/virt-who-config-{}.conf"
+        .format(config_id)
+    )
+
+
+def get_configure_option(option, filename):
+    """Return the option's value for the specific file.
+    :param str option: the option name in the configure file
+    :param str filename: the configure file, it could be:
+        /etc/sysconfig/virt-who
+        /etc/virt-who.d/virt-who-config-{}.conf
+    :raises: VirtWhoError: If no this option name in the file.
+    """
+    cmd = "grep -v '^#' {} |grep ^{}".format(filename, option)
```

Comment:
nitpick, but can you add a space between | and grep?

Reply:
Nice